### DISCUSSION TITLE
setExpirationTime format

### DISCUSSION THREAD

**Discussion by bradisbell** [panva/jose#373] [Q&A] [answered]
The documentation (https://github.com/panva/jose/blob/8d016fdb5825dc0610707b142e898981bf1e5a47/docs/classes/jwt_sign.SignJWT.md#setexpirationtime) doesn't specify the format for strings.  I see in the example, there is something like `2h` for 2 hours.  What is the format for these strings?

**Comment by panva** [comment, marked as answer]
seconds, minutes, hours, days, weeks, years and common abbreviations of those, if you get it wrong it'll tell you. 

https://github.com/panva/jose/blob/v4.5.1/src/lib/secs.ts

**Reply by bradisbell** [reply]
Woot, thanks!
